shared/checks/oval/audit_rules_media_export.xml

```diff
@@ -31,13 +31,17 @@
       </criteria>
     </criteria>
   </definition>
-
+  {{% if product == "rhel6" %}}
+    {{% set AUID = 500 %}}
+  {{% else %}}
+    {{% set AUID = 1000 %}}
+  {{% endif %}}
```

Comment:
this is repeated so much that I think we should somehow share the code. We could have `auid: 1000` in product.yml or create a jinja2 macro.

Reply:
That is a good idea, I have proposed a solution in #3114. Once merged I'll updated this PR.

Reply:
This has been updated to use the new `auid` value.